djangosaml2/backends.py

```diff
@@ -188,7 +188,7 @@ def _get_saml2_user(self, main_attribute, attributes, attribute_mapping):
             user = User.objects.get(**user_query_args)
             user = self.update_user(user, attributes, attribute_mapping)
         except User.DoesNotExist:
-            logger.error('The user "%s" does not exist', main_attribute)
+            logger.error('The user "%s" does not exist, searched %s', (main_attribute, django_user_main_attribute))
```

Comment:
I believe those 2 logger arguments at the end should not be wrapped inside a tuple. Please fix.
Otherwise OK :+1: